docs/developing/routing.md

```diff
@@ -0,0 +1,12 @@
+# Routing
+
+## All Roads Lead To Rome
+
+There are many roads that lead to Rome, and different apps have different requirements. For this reason, it's hard to recommend a single router for your project. Instead, we'll provide you with a list of different routers for different usecases that work well with your web component projects.
+
+- [pwa-helpers router](https://github.com/Polymer/pwa-helpers#routerjs)
+- [redux first routing](https://github.com/mksarge/redux-first-routing)
+- [lit-element-router](https://github.com/hamedasemi/lit-element-router)
+- [lit-router](https://github.com/danevans/lit-router)
+- [vaadin-router](https://github.com/vaadin/vaadin-router)
```

Comment:
Shall we order these by recommendation, or order them alphabetically (and mention it) to express to preference?

Reply:
I'd say order alphabetically